libmscore/element.cpp

```diff
@@ -312,6 +312,20 @@ Fraction Element::rtick() const
       return Fraction(0, 1);
       }
 
+//---------------------------------------------------------
+//   beat
+//---------------------------------------------------------
+
+float Element::beat() const
```

Comment:
Fraction could be a better choice because of rounding and use in comparisons, see also below.